src/Indice.GeoIP/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Indice.GeoIP.GeoLite2;
+using Indice.GeoIP.Services;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Configuration;
```

Comment:
change namespace to `Microsoft.Extensions.DependencyInjection` and the class and folder name to `GeoIPFeatureExtensions`